santosfamilyfoundation/RuinaMain
# [RUIN-194], [RUIN-203] Add diagram screen and ability to save crash photo for report

# Description

Having an ability to capture what a crash looks like is crucial to understanding how a crash occurred. The previous version of the app did not have a place to capture this data. Now when the user is on the home screen, they have the ability to add a photo of the crash.

# Images
![image](https://user-images.githubusercontent.com/42981026/151382434-ec7d27a9-44d9-4496-aa74-24807acc5682.png)

# Testing

1. Start New Report
2. Fill out Quick Survey
3. Click "Add Crash Diagram"
4. Take a photo
5. Check that image appears below the button
6. Force close the app
7. Continue Existing Report
8. Check that reports successfully loads
9. Check that image appears and is the same image
